### PR TITLE
enable renderTab property.

### PR DESCRIPTION
**Pull request by code-artisan**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [x] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [x] Add unit tests for the feature.

### CODE REVIEW

**Comment by bang88** [comment]
Please update related document and fixes tests.

**Comment by code-artisan** [comment]
@bang88 The document is ok. Just you ignore `renderTab` property.

**Comment by bang88** [comment]
My bad. Please fixes ci error so we can merge it ASAP.

File: components/tabs/Tabs.tsx [hunk 1]
```diff
@@ -358,6 +358,7 @@ export class Tabs extends React.PureComponent<TabsProps, StateType> {
       tabBarPosition,
       tabBarTextStyle,
       tabBarUnderlineStyle,
+      renderTab,
```
Comment (bang88) [inline components/tabs/Tabs.tsx]:
```ts
  /** render the tab of tabbar */
  renderTab?: (tab: TabData) => React.ReactNode;
```
`renderTab` does not exists on `Tabs` Props , need add it to `PropsType`

**Pull request closed without merging**